modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc

```diff
@@ -11,67 +11,85 @@ The workspace engine and authentication system used up to and including {prod-pr
 
 .Prerequisites
 
-* An instance of {prod-prev-short} deployed on xref:supported-platforms.adoc[], using the default internal PostgreSQL database, and with OpenShift OAuth enabled. See {prod-docs-url-enable-oauth}.
-* Command line tools `oc`, `curl`, and `jq` are available.
+* An instance of {prod-prev-short} deployed on one of the xref:supported-platforms.adoc[], using the default internal PostgreSQL database, and with OpenShift OAuth enabled. See {prod-docs-url-enable-oauth}.
```

Comment:
```suggestion
* An instance of {prod-prev-short} running on one of the xref:supported-platforms.adoc[]. This instance uses the default internal PostgreSQL database and has OpenShift OAuth enabled. See {prod-docs-url-enable-oauth}.
```

Reply:
@rolfedh, i applied it with a couple of minor tweaks:
```
* An instance of {prod-prev-short} deployed on one of the xref:supported-platforms.adoc[]. The instance uses the default internal PostgreSQL database and has OpenShift OAuth enabled. See {prod-docs-url-enable-oauth}. 
```